configure.ac

```diff
@@ -1262,6 +1262,8 @@ AS_CASE([$host],
     [arch=amd64; system=win64],
   [[powerpc64le*-*-linux*]],
     [has_native_backend=yes; arch=power; model=ppc64le; system=linux],
+  [[powerpc64*-*-linux-musl*]],
```

Comment:
Right.  For the time being, I just added a check on _CALL_ELF in runtime/power.S, to abort in case of ABI mismatch.  We'll see later if this needs to be done in configure.